website/cue/reference/remap/functions/is_empty.cue

```diff
@@ -0,0 +1,49 @@
+package metadata
+
+remap: functions: is_empty: {
+	category: "Type"
+	description: """
+		Check if the object, array, or string is empty.
```

Comment:
```suggestion
		Check if the object, array, or string contains any elements.
```

Just thinking using the word empty again in the description is a little redundant given it is already in the name.